File: bubblesort.c
#include<stdio.h>
void main()
{ 
int i=0,j,n,temp;
printf("enter no. of elements  to be inputed in array:");
scanf("%d",&n);
int arr[n];

for(int i=0;i<n;i++){
printf("enter element at index %d\n",i+1);
scanf("%d",&arr[i]);}

for(i=0;i<n;i++){
for(j=0;j<n;j++){
if(arr[j]>arr[j+1]){ 

temp=arr[j];
arr[j]=arr[j+1];
arr[j+1]=temp;
}}}

printf(" sorted array is\n");
for(int i=0;i<n;i++){
printf("%d\n",arr[i]);
}
}

File: bubblesort.py
#this code has the ability to coduct bubble sort algorithm on int values only
print("enter no. of values to be inputed")
n=int(input())

l=[]
print("enter elements in array:\n")
for i in range(0,n):
    ele=input()
    l.append(ele)
print("unsorted elements are\n",l)
for j in range(0,n-1):
    for i in range(0,n-1):
        if(l[i]>l[i+1]):
            temp = l[i]
            l[i] = l[i+1]
            l[i+1] = temp
print("sorted array is:\n",l)
